relays/client-rococo/src/runtime.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Types that are specific to the Rococo runtime.
+
+use bp_messages::{LaneId, UnrewardedRelayersState};
+use bp_polkadot_core::PolkadotLike;
+use bp_runtime::Chain;
+use codec::{Decode, Encode};
+use frame_support::weights::Weight;
+
+/// Instance of messages pallet that is used to bridge with Wococo chain.
+pub type WithWococoMessagesInstance = pallet_bridge_messages::Instance1;
+
+/// Unchecked Rococo extrinsic.
+pub type UncheckedExtrinsic = bp_polkadot_core::UncheckedExtrinsic<Call>;
+
+/// Wococo account ownership digest from Rococo.
+///
+/// The byte vector returned by this function should be signed with a Wococo account private key.
+/// This way, the owner of `rococo_account_id` on Rococo proves that the Wococo account private key
+/// is also under his control.
+pub fn rococo_to_wococo_account_ownership_digest<Call, AccountId, SpecVersion>(
+	wococo_call: &Call,
+	rococo_account_id: AccountId,
+	wococo_spec_version: SpecVersion,
+) -> Vec<u8>
+where
+	Call: codec::Encode,
+	AccountId: codec::Encode,
+	SpecVersion: codec::Encode,
+{
+	pallet_bridge_dispatch::account_ownership_digest(
+		wococo_call,
+		rococo_account_id,
+		wococo_spec_version,
+		bp_runtime::ROCOCO_CHAIN_ID,
+		bp_runtime::WOCOCO_CHAIN_ID,
+	)
+}
+
+/// Rococo Runtime `Call` enum.
+///
+/// The enum represents a subset of possible `Call`s we can send to Rococo chain.
+/// Ideally this code would be auto-generated from Metadata, because we want to
+/// avoid depending directly on the ENTIRE runtime just to get the encoding of `Dispatchable`s.
+///
+/// All entries here (like pretty much in the entire file) must be kept in sync with Rococo
+/// `construct_runtime`, so that we maintain SCALE-compatibility.
+///
+/// See: https://github.com/paritytech/polkadot/blob/master/runtime/rococo/src/lib.rs
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone)]
+pub enum Call {
```

Comment:
The reason why I initially envisioned that in `primitives` is to make it easy for the other runtime to actually import `primitives` and create this call from a pallet in a (near) future.
That said, getting the testnet out is much higher priority than imaginary future use cases :)